packages/paste-theme/src/usePasteHighchartsTheme.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import {ThemeContext} from '@twilio-paste/styling-library';
+import type {ThemeShape} from './types/themes';
+
+type HighchartsOptions = {
+  colors: string[];
+};
+
+// Needs more iteration, but currently returns the colors array used when
+// setting a theme in highCharts. See https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/members/setoptions
+const usePasteHighchartsTheme = (): HighchartsOptions => {
```

Comment:
Is there anything specific to hightcharts here? I'm wondering if we're too tied to Hightcharts, could we get away with a more generic name?

Reply:
OK, yeah, part one answers part two for me then. 

Based on justification of the second, I think I tend to agree with you there. Because it's specific to highcharts, not putting it into the theme package probably makes sense there. If we switch libraries, we can just drop the highcharts package as a separate entity